packages/backend/src/database/migrations/20220329153255_themes.ts

```diff
@@ -0,0 +1,16 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.raw(`
+CREATE TABLE IF NOT EXISTS themes (
```

Comment:
Looking at the query pattern for this in the model (either create or get by org id), I'm not sure if we need the extra table. We could have just a `chart_colors TEXT [] NULLABLE` on the `organizations` table. Any downside to doing that? It would keep things simpler, and we don't have to maintain a 1:1 relationship in the database.

Also for variables, we prefer US spelling for everything `colors` not `colours`